guides/v2.1/test/unit/unit_test_execution_cli.md

```diff
@@ -50,7 +50,7 @@ php -f vendor/bin/phpunit -- -c dev/tests/unit/phpunit.xml.dist
 ```
 
 #### Use the correct PHP interpreter
-On many development systems, you might have more one PHP version installed. You must know the correct PHP interpreter to use for testing; that is, Buse the same version of PHP to run the unit tests you use to run Magento.
```

Comment:
This commit seems to be a duplicate of the one in PR: #2028

Reply:
@mhauri : Ther are 2 files that were affected. #2028 points to only one file.